play/src/front/Connection/RoomConnection.ts

```diff
@@ -758,16 +757,24 @@ export class RoomConnection implements RoomConnection {
                     this._askMutedVideoMessage.next(message.askMutedVideoMessage);
                     break;
                 }
-                case "proximityPrivateMessageToClientMessage": {
-                    this._proximityPrivateMessageToClientMessageStream.next(
-                        message.proximityPrivateMessageToClientMessage
-                    );
+                case "publicEvent": {
+                    switch (message.publicEvent.spaceEvent?.event?.$case) {
+                        case "spaceMessage": {
+                            this._proximityPublicMessageEvent.next(message.publicEvent);
+                            break;
+                        }
+                        case "spaceIsTyping": {
+                            this._typingProximityEvent.next(message.publicEvent);
+                            break;
+                        }
+                        default: {
+                            break;
```

Comment:
```suggestion
                            const _exhaustiveCheck: never = message.publicEvent.spaceEvent?.event;
```

With this, an error will be raised if we forget to handle a public event properly in the front